packages/action-tracker/src/index.ts

```diff
@@ -32,13 +47,23 @@ export const buildEmbraceMiddleware = () => {
           );
         } else {
           const endTime = new Date().getTime();
-          NativeModules.EmbraceManager.logRNAction(
-            action.type.toString().toUpperCase(),
+          attributePromises.push(
+            NativeModules.EmbraceManager.addSpanAttributeToSpan(
+              spanId,
+              "outcome",
+              "SUCCESS",
+            ),
+          );
+
+          await Promise.all(attributePromises);
```

Comment:
I think we're better off just firing these off and not waiting on them and if something does go wrong we wouldn't want to block the app because of this instrumentation

Reply:
but, If we do that, we cannot guarantee that the stop span didnt finish first than the other promises. Can a stopped span received new attributes ?

Reply:
nope a stopped span becomes readonly so new attributes would be ignored

I think it's an unlikely enough case that I'd be fine with the simpler fire and forget approach here, but if we are worried about it I should confirm my assumption that this would actually have an impact on the user's code. Do you know how Redux's middleware management would handle this having an async action? Would it wait on it or would it just continue on to the next middleware?

Reply:
rest of the changes lgtm after testing and resolving this ^ question